HunterJuneau/dine-and-dash
# Cm add projects

## Description

Admin is able to add/create a new product, edit a product, and change availability (soft delete).
Admin can see all products, even those not available. A normal user can only see available products.
Some very basic styling added

## Related Issue
#76 
#73 
#74 
#60 

## Motivation and Context

This allow the user to only purchase available products and the Admin can control the inventory.

## How Can This Be Tested?

Add, Edit, change availability of a products. Sign in as an admin and regular user to see the difference in product views.

## Types of changes
